rasa/nlu/featurizers/sparse_featurizer/regex_featurizer.py

```diff
@@ -42,6 +46,9 @@ def required_components(cls) -> List[Type[Component]]:
         "use_lookup_tables": True,
         # use regexes to generate features
         "use_regexes": True,
+        # Additional number of patterns to consider
+        # for incremental training
+        "number_additional_patterns": None,
```

Comment:
Ahh no, because `RegexEntityExtractor` is more like an online extractor and does not compute any feature vectors. Since it just loads up all the patterns present in the data at that point and persists them for inferencing we don't need to account for additional patterns.